math/binarymath/arithmeticmean.go

```diff
@@ -0,0 +1,15 @@
+// arithmeticmean.go
+// description: Arithmetic mean
+// details:
+// The most common type of average is the arithmetic mean. If n numbers are given, each number denoted by ai (where i = 1,2, ..., n), the arithmetic mean is the sum of the as divided by n or - [Arithmetic mean](https://en.wikipedia.org/wiki/Average#Arithmetic_mean)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see arithmeticmean_test.go
+package binarymath
```

Comment:
Could you add a space between these lines or in the test file - if you go to the [godoc](https://pkg.go.dev/github.com/TheAlgorithms/Go) you will see that these comments are parsed as package description. If multiple files have package documentation then both get appended and the order is also not guaranteed.

```suggestion
// arithmeticmean.go
// description: Arithmetic mean
// details:
// The most common type of average is the arithmetic mean. If n numbers are given, each number denoted by ai (where i = 1,2, ..., n), the arithmetic mean is the sum of the as divided by n or - [Arithmetic mean](https://en.wikipedia.org/wiki/Average#Arithmetic_mean)
// author(s) [red_byte](https://github.com/i-redbyte)
// see arithmeticmean_test.go

// Package binarymath describes algorithms that use binary operations for different calculations.
package binarymath
```



<img width="1357" alt="Screenshot 2021-09-23 at 18 12 50" src="https://user-images.githubusercontent.com/21105489/134553249-cd5d38e3-d5bb-4f34-87c0-a936cf526733.png">

Reply:
Done